addons/tasks/CfgVehicles.hpp

```diff
@@ -40,6 +40,12 @@ class CfgVehicles
                       class opt_3 {name = "Debug"; value = 2;};
                   }
               }
+              class TaskRequestItems {
+                  displayName = "Task Request Item";
+                  description = "The items where the task request action should be applied to";
```

Comment:
add the ``["doc1", "doc2"]`` example here, like ALiVE modules do. This will make it clear if the user needs to type in the [ ] and if the variables inside it must have " ".

Reply:
done and done